e2e-tests/IIoTPlatform-E2E-Tests/Standalone/B_PublishMultipleNodesStandaloneDirectMethodTestTheory.cs

```diff
@@ -175,8 +198,23 @@ IIoTMultipleNodesTestContext context
             Assert.Equal((int)HttpStatusCode.OK, response.Status);
 
             // Wait till the publishing has stopped.
+            await Task.Delay(TestConstants.DefaultTimeoutInMilliseconds, cts.Token).ConfigureAwait(false);
+            await Task.Delay(TestConstants.DefaultTimeoutInMilliseconds, cts.Token).ConfigureAwait(false);
```

Comment:
>await Task.Delay [](http://example.com/codeflow?start=12&length=16)

why do we do this 3 times? could we maybe do it once
3 * TestConstants.DefaultTimeoutInMilliseconds

Reply:
Do we actually need three of them ? Would one maybe suffice ?

Reply:
removed. One delay is enough